styles/templates/_defaultElementOwl.scss

```diff
@@ -0,0 +1,182 @@
+.ontodia-default-owl-template {
```

Comment:
Please rename this file to `_defaultOwl.scss` to match component's name.

Reply:
done